### ISSUE TITLE
"None" in email

### ISSUE THREAD

**Issue by jparmstrong** [nfultz/nb2mail#13]
Hello! I've come across bug with the recent versions of nbconvert that causes None to appear on the email. 

It's because of the mail.tpl extends a deprecated `display_priority.tpl` file, it should be `base/display_priority.j2`

**Comment by nfultz** [comment]
Thank you for your note. 

I'm starting a second job teaching in addition to my normal full time job, so I probably can't spend too much time on this or push a new release before spring break.

There is this note on the current man page that may be a workaround for you:

```
Be aware that if you were using custom copies of the old 5.x template files (i.e. --template), you will now need to use --template-file path/to/old/file.tpl in order to use that file in compatibility mode as opposed to other options.
```

If this is urgent for you a PR would be appreciated.

A couple notes after looking into this for 15 minutes:

* nbconvert changelog: https://nbconvert.readthedocs.io/en/latest/changelog.html
     * They dropped Python2 in 6.0.0
         * nb2mail has supported Python2 this whole time, but no reason to if nbconvert doesn't
     * nbconvert changed template system to jinja 
         * Based on that, refactor whole mail template? Note sure if a .tpl template can extend a .j2 template. 
     * Despite years of discussion, PostProcessors still not deprecated  - https://github.com/jupyter/nbconvert/issues/582

**Comment by jparmstrong** [comment]
Thanks for looking into it @nfultz ! I went ahead and raised a PR to address the issue.

**Comment by nfultz** [comment]
Fixed with the 1.0 release
